docs/docs/transforms/label.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: transform
+title: Label Transform
+permalink: /docs/transforms/label/index.html
+---
+
+The **label** transform repositions texts in text mark, so that their placements are not colliding with other elements in the chart. Those elements include marks that have name included in property `avoidMarks` and the mark that is used as backing data (as explained in [reactive geometry](../../marks/#reactivegeom)) of the text mark this label transform is transforming.
+
+The label transform is useful for labeling data points by creating a text mark that takes in data from the mark that represents the data point (we will call this 'base mark' for this label documentation), then use the label transform on the text mark to reposition the text, so that the texts appear near their data points without colliding into other objects in the chart.
```

Comment:
```suggestion
The label transform is useful for labeling data points by creating a text mark that takes in data from the mark that represents the data point (we will call this mark the "base mark"). You can then use the label transform on the text mark to reposition the text so that the texts appear near their data points without colliding with other objects in the chart.
```